### PR TITLE
Gui tests 2642

### PR DESCRIPTION
**Pull request by maryvictol**
This PR adds e2e tests for Restrict a count of the instances to launch #2642

### CODE REVIEW

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/ao/RunsMenuAO.java [hunk 1]
```diff
@@ -393,6 +393,11 @@ public RunsMenuAO resume(final String runId, final String pipelineName) {
         return this;
     }
 
+    public PipelineRunFormAO rerun(final String runId, final String pipelineName) {
```
Comment (kamyshova) [inline e2e/gui/src/test/java/com/epam/pipeline/autotests/ao/RunsMenuAO.java]: Parameter 'pipelineName' is not used

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/ao/SettingsPageAO.java [hunk 2]
```diff
@@ -1051,8 +1072,28 @@ public GroupsTabAO ok() {
                     return parentAO;
                 }
 
+                public EditGroupPopup addUser(final Account name) {
+                    click(SEARCH);
+                    actions().sendKeys(user.login).perform();
```
Comment (kamyshova) [inline e2e/gui/src/test/java/com/epam/pipeline/autotests/ao/SettingsPageAO.java]: I suppose we should pass the `name` value instead of `user.login`

**Review by kamyshova — CHANGES_REQUESTED** [review]

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/ao/RunsMenuAO.java (hunk 1 above)
Comment (maryvictol) [reply e2e/gui/src/test/java/com/epam/pipeline/autotests/ao/RunsMenuAO.java]: Extra parameter 'pipelineName has been removed

File: e2e/gui/src/test/java/com/epam/pipeline/autotests/ao/SettingsPageAO.java (hunk 2 above)
Comment (maryvictol) [reply e2e/gui/src/test/java/com/epam/pipeline/autotests/ao/SettingsPageAO.java]: Done

**Pull request closed without merging**